src/tools/genpolicy/genpolicy-settings.json

```diff
@@ -282,8 +282,18 @@
             "cc-managed-premium-csi"
         ],
         "smb_storage_classes": [
-            "cc-azurefile-csi",
-            "cc-azurefile-premium-csi"
+            {
+                "name": "azurefile-csi-kata-cc",
```

Comment:
We probably want to use `azurefile-csi` and `azurefile-premium-csi` since we want the official driver to support conf pods?

Reply:
The default storage drivers do not have enableKataCCMount set to true.

Reply:
I agree with @sprt, it would be good to include the default Azure File storage classes. For example, our team creates a custom storage class with name `azurefile-csi` and set the `enableKataCCMount` flag to true; we use the same name for compatibility with non-confidential deployments. Adding them would also align with the plan of removing the flag from the storage class and automatically enabling kata mount for confidential pods in the Azure File CSI driver.